blackjax/sgmcmc/diffusion.py

```diff
@@ -35,6 +35,102 @@ def one_step(
 
     return one_step
 
+""" 
+    We are interested in the simulations of exp(-U(x) / T), 
+    where U is energy function and T is the temperature.
+
+    logprob normally means U(x)/ T instead of U(x).
+"""
+
+
+class ContourState(NamedTuple):
```

Comment:
Would it be possible to add a docstring that explains what `energy_pdf` and `energy_idx` correspond to? The other two are self-explanatory.

Reply:
No problem, I will add an explanation very soon.